src/Behat/Gherkin/Node/FeatureNode.php

```diff
@@ -254,13 +254,11 @@ public function getLine()
      * @param string $file A file path
      *
      * @return bool
-     *
-     * @see https://github.com/symfony/filesystem/blob/master/Filesystem.php
```

Comment:
I think it might be better to keep this comment a) to give at least a bit of context (it's hard looking at the code to understand what it's doing, at least knowing where it came from provides a pointer to go and find history/tests/etc if ever needed) and b) to provide some credit to the Symfony project given this appears to be copy/pasted?